src/Farmer/Builders/Builders.Functions.fs

```diff
@@ -159,6 +275,17 @@ type FunctionsConfig =
             | Some _
             | None ->
                 ()
+                
+            for kvp in this.Slots do
+                let name,cfg = kvp.Key,kvp.Value
+                { SlotName = name 
+                  Location = location
+                  ServicePlan = this.ServicePlanId
+                  Site = this.ResourceId
+                  Tags = this.Tags
```

Comment:
Should we allow slots to have their own tags? and if so, do we merge with the web app slot tags?

Reply:
I don't think at this point in the code we should make a decision like that. Would that be a codat thing or do you think it just makes sense in general?